functions/poll.js

```diff
@@ -439,11 +471,14 @@ async function pollTable(table, statusToCallbacks) {
     // and update its meta field
     const updatedMeta = _.assign(meta, _.reduce(_.map(results, 'value'), _.assign));
     updatedMeta.lastSeenStatus = fields.status || null;
+
     console.log('updatedMeta', updatedMeta);
-    return await updateRecord(table, id, {}, updatedMeta);
+    await updateRecord(table, id, {}, updatedMeta);
+    return null;
```

Comment:
is there a reason we're returning null everywhere now? when working with async/await and promises, especially when things are nested or chained together, it's best practice to return the promise/return the await